# SPDX-FileCopyrightText: 2026 Aleix Pol Gonzalez <aleix.pol@codethink.co.uk>
# SPDX-License-Identifier: BSD-2-Clause

diff --git a/elements/sdk/gcr.bst b/elements/sdk/gcr.bst
index b6d15420d..aac6a3c7f 100644
--- a/elements/sdk/gcr.bst
+++ b/elements/sdk/gcr.bst
@@ -16,7 +16,7 @@ build-depends:
 
 depends:
 - sdk/glib.bst
-- sdk/gtk.bst
+- sdk/gtk+-3.bst
 - sdk/libsecret.bst
 - freedesktop-sdk.bst:components/gnupg.bst
 - freedesktop-sdk.bst:components/openssh.bst
@@ -24,6 +24,10 @@ depends:
 - freedesktop-sdk.bst:components/systemd-libs.bst
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
+variables:
+  meson-local: >-
+    -Dgtk4=false
+
 public:
   bst:
     split-rules:
